### PR TITLE
Add CountdownTimer.increment read-only property

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

**Comment by cbracken** [comment]
HE'S ALIIIIVE! Thanks!

File: lib/src/async/countdown_timer.dart [hunk 1]
```diff
@@ -24,8 +24,9 @@ class CountdownTimer extends Stream<CountdownTimer> {
   static const _THRESHOLD_MS = 4;
 
   final Duration _duration;
-  final Duration _increment;
   final Stopwatch _stopwatch;
+
+  final Duration increment;
```
Comment (yjbanov) [inline lib/src/async/countdown_timer.dart]: Dart docs? Sorry for jumping in too late.

Comment (cbracken) [inline lib/src/async/countdown_timer.dart]: PR #318

**Pull request merged**